test/rai/pipeline_scorecard.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
```

Comment:
These two YAMLs don't appear to be hooked into the regular testing infrastructure?

Reply:
added the component in the existing boston yaml pipeline now.